### ISSUE TITLE
Could not pass event PlayerInteractEvent to PlotSquared v4.13

### ISSUE THREAD

**Issue by andris155** [IntellectualSites/PlotSquared#2258]
# Bug report template 

**Debug paste link:** 

https://incendo.org/paste/view/136e919fb9c646daaa8a6cded5389c0c

**Description of the problem:** 
Console error
```
[13:56:33] [Server thread/ERROR]: Could not pass event PlayerInteractEvent to PlotSquared v4.13
org.bukkit.event.EventException: null
	at org.bukkit.plugin.EventExecutor$2.execute(EventExecutor.java:72) ~[patched_1.13.2.jar:git-Paper-507]
	at co.aikar.timings.TimedEventExecutor.execute(TimedEventExecutor.java:78) ~[patched_1.13.2.jar:git-Paper-507]
	at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:62) ~[patched_1.13.2.jar:git-Paper-507]
	at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:513) ~[patched_1.13.2.jar:git-Paper-507]
	at org.bukkit.craftbukkit.v1_13_R2.event.CraftEventFactory.callPlayerInteractEvent(CraftEventFactory.java:315) ~[patched_1.13.2.jar:git-Paper-507]
	at net.minecraft.server.v1_13_R2.PlayerInteractManager.a(PlayerInteractManager.java:474) ~[patched_1.13.2.jar:git-Paper-507]
	at net.minecraft.server.v1_13_R2.PlayerConnection.a(PlayerConnection.java:1361) ~[patched_1.13.2.jar:git-Paper-507]
	at net.minecraft.server.v1_13_R2.PacketPlayInUseItem.a(PacketPlayInUseItem.java:37) ~[patched_1.13.2.jar:git-Paper-507]
	at net.minecraft.server.v1_13_R2.PacketPlayInUseItem.a(PacketPlayInUseItem.java:5) ~[patched_1.13.2.jar:git-Paper-507]
	at net.minecraft.server.v1_13_R2.PlayerConnectionUtils.lambda$ensureMainThread$0(PlayerConnectionUtils.java:13) ~[patched_1.13.2.jar:git-Paper-507]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515) ~[?:?]
	at java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at net.minecraft.server.v1_13_R2.SystemUtils.a(SystemUtils.java:108) ~[patched_1.13.2.jar:git-Paper-507]
	at net.minecraft.server.v1_13_R2.MinecraftServer.b(MinecraftServer.java:1011) ~[patched_1.13.2.jar:git-Paper-507]
	at net.minecraft.server.v1_13_R2.DedicatedServer.b(DedicatedServer.java:439) ~[patched_1.13.2.jar:git-Paper-507]
	at net.minecraft.server.v1_13_R2.MinecraftServer.a(MinecraftServer.java:938) ~[patched_1.13.2.jar:git-Paper-507]
	at net.minecraft.server.v1_13_R2.MinecraftServer.run(MinecraftServer.java:836) ~[patched_1.13.2.jar:git-Paper-507]
	at java.lang.Thread.run(Thread.java:834) [?:?]
Caused by: java.lang.NullPointerException
	at com.github.intellectualsites.plotsquared.bukkit.listeners.PlayerEvents.onInteract(PlayerEvents.java:1882) ~[?:?]
	at com.destroystokyo.paper.event.executor.asm.generated.GeneratedEventExecutor49.execute(Unknown Source) ~[?:?]
	at org.bukkit.plugin.EventExecutor$2.execute(EventExecutor.java:70) ~[patched_1.13.2.jar:git-Paper-507]
	... 17 more
```

**How to replicate:** 

**Checklist**:

- [x] I included a `/plot debugpaste` link
- [x] I made sure there are no duplicates of this report [(Use Search)](https://github.com/IntellectualSites/PlotSquared/issues?utf8=%E2%9C%93&q=is%3Aissue)
- [x] I made sure I am using an up-to-date version of PlotSquared
- [x] I made sure the bug/error is not caused by any other plugin

**Comment by AeSix** [comment]
Is this still an issue with newer builds?
https://ci.athion.net/view/Everything/job/PlotSquared-Breaking/

**Comment by andris155** [comment]
Yes
```
[16:12:52] [Server thread/ERROR]: Could not pass event PlayerInteractEvent to PlotSquared v4.15
org.bukkit.event.EventException: null
	at org.bukkit.plugin.EventExecutor$2.execute(EventExecutor.java:72) ~[patched_1.13.2.jar:git-Paper-518]
	at co.aikar.timings.TimedEventExecutor.execute(TimedEventExecutor.java:78) ~[patched_1.13.2.jar:git-Paper-518]
	at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:62) ~[patched_1.13.2.jar:git-Paper-518]
	at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:513) ~[patched_1.13.2.jar:git-Paper-518]
	at org.bukkit.craftbukkit.v1_13_R2.event.CraftEventFactory.callPlayerInteractEvent(CraftEventFactory.java:320) ~[patched_1.13.2.jar:git-Paper-518]
	at net.minecraft.server.v1_13_R2.PlayerInteractManager.a(PlayerInteractManager.java:466) ~[patched_1.13.2.jar:git-Paper-518]
	at net.minecraft.server.v1_13_R2.PlayerConnection.a(PlayerConnection.java:1362) ~[patched_1.13.2.jar:git-Paper-518]
	at net.minecraft.server.v1_13_R2.PacketPlayInUseItem.a(PacketPlayInUseItem.java:37) ~[patched_1.13.2.jar:git-Paper-518]
	at net.minecraft.server.v1_13_R2.PacketPlayInUseItem.a(PacketPlayInUseItem.java:5) ~[patched_1.13.2.jar:git-Paper-518]
	at net.minecraft.server.v1_13_R2.PlayerConnectionUtils.lambda$ensureMainThread$0(PlayerConnectionUtils.java:15) ~[patched_1.13.2.jar:git-Paper-518]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515) ~[?:?]
	at java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at net.minecraft.server.v1_13_R2.SystemUtils.a(SystemUtils.java:108) ~[patched_1.13.2.jar:git-Paper-518]
	at net.minecraft.server.v1_13_R2.MinecraftServer.b(MinecraftServer.java:1011) ~[patched_1.13.2.jar:git-Paper-518]
	at net.minecraft.server.v1_13_R2.DedicatedServer.b(DedicatedServer.java:439) ~[patched_1.13.2.jar:git-Paper-518]
	at net.minecraft.server.v1_13_R2.MinecraftServer.a(MinecraftServer.java:938) ~[patched_1.13.2.jar:git-Paper-518]
	at net.minecraft.server.v1_13_R2.MinecraftServer.run(MinecraftServer.java:836) ~[patched_1.13.2.jar:git-Paper-518]
	at java.lang.Thread.run(Thread.java:834) [?:?]
Caused by: java.lang.NullPointerException
	at com.github.intellectualsites.plotsquared.bukkit.listeners.PlayerEvents.onInteract(PlayerEvents.java:1880) ~[?:?]
	at com.destroystokyo.paper.event.executor.asm.generated.GeneratedEventExecutor48.execute(Unknown Source) ~[?:?]
	at org.bukkit.plugin.EventExecutor$2.execute(EventExecutor.java:70) ~[patched_1.13.2.jar:git-Paper-518]
	... 17 more
```

**Comment by dordsor21** [comment]
Ought to be fixed in 8710f2f
